.github/workflows/run-tests.yml

```diff
@@ -42,6 +42,25 @@ jobs:
           path: "**/target/surefire-reports/TEST-*.xml"
           retention-days: 10
 
+  integration-h2-test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          java-version: 11
+          distribution: temurin
+          cache: maven
+      - name: Run integration h2 tests
+        run: mvn --threads 2C test --batch-mode --no-transfer-progress -PintegrationH2 -f ./dhis-2/pom.xml -pl -dhis-web-embedded-jetty
+      - name: Archive surefire reports
+        uses: actions/upload-artifact@v2
+        with:
+          name: surefire-reports
```

Comment:
We don't need them anymore. I wanted to remove the other surefire report archive steps as well.

Reply:
I forgot to say 😬 : unless you want to use the xml reports for further analysis.

Reply:
I don't need them.
Not sure if someone needs them though